dagger1support/src/main/java/mortar/dagger1support/ObjectGraphService.java

```diff
@@ -0,0 +1,103 @@
+package mortar.dagger1support;
+
+import android.app.Activity;
+import android.content.Context;
+import dagger.ObjectGraph;
+import java.util.Collection;
+import mortar.MortarScope;
+import mortar.bundler.BundleServiceRunner;
+
+/**
+ * Provides utility methods for using Mortar with Dagger 1.
+ */
+public class ObjectGraphService {
+  public static final String SERVICE_NAME = ObjectGraphService.class.getName();
+
+  /**
+   * Create a new {@link ObjectGraph} based on the given module. The new graph will extend
+   * the graph found in the parent scope (via {@link ObjectGraph#plus}), if there is one.
+   */
+  public static ObjectGraph create(MortarScope parent, Object... daggerModules) {
+    ObjectGraph parentGraph = getObjectGraph(parent);
+
+    return parentGraph == null ? ObjectGraph.create(daggerModules)
+        : parentGraph.plus(daggerModules);
+  }
+
+  public static ObjectGraph getObjectGraph(Context context) {
+    return (ObjectGraph) context.getSystemService(ObjectGraphService.SERVICE_NAME);
+  }
+
+  public static ObjectGraph getObjectGraph(MortarScope scope) {
+    return scope.getService(ObjectGraphService.SERVICE_NAME);
+  }
+
+  /**
+   * A convenience wrapper for {@link ObjectGraphService#getObjectGraph} to simplify dynamic
+   * injection, typically for {@link Activity} and {@link android.view.View} instances that must be
+   * instantiated by Android.
+   */
+  public static void inject(Context context, Object object) {
+    getObjectGraph(context).inject(object);
+  }
+
+  private static ObjectGraph createSubgraphBlueprintStyle(ObjectGraph parentGraph,
+      Object daggerModule) {
+    ObjectGraph newGraph;
+    if (daggerModule == null) {
+      newGraph = parentGraph.plus();
+    } else if (daggerModule instanceof Collection) {
+      Collection c = (Collection) daggerModule;
+      newGraph = parentGraph.plus(c.toArray(new Object[c.size()]));
+    } else {
+      newGraph = parentGraph.plus(daggerModule);
+    }
+    return newGraph;
+  }
+
+  /**
+   * Returns the existing {@link MortarScope} scope for the given {@link Activity}, or
+   * uses the {@link Blueprint} to create one if none is found. The scope will provide
+   * {@link mortar.bundler.BundleService} and {@link BundleServiceRunner}.
+   * <p/>
+   * It is expected that this method will be called from {@link Activity#onCreate}. Calling
+   * it at other times may lead to surprises.
+   */
+  @Deprecated public static MortarScope requireActivityScope(MortarScope parentScope,
```

Comment:
Add a `@deprecated` note in the javadoc, explaining when and why and/or what to do instead.